.github/services/sftp/sftp-default-root/action.yml

```diff
@@ -0,0 +1,47 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: sftp-default-root
+description: 'Behavior test for SFTP with default root'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup sftp
+      shell: bash
+      run: |
+        mkdir -p target/ssh
```

Comment:
Is it possible to finish those setup in docker-compose or Dockerfile? I find it's quiet hard to reproduce locally.

Also cc @silver-ymz

Reply:
Let's merge this PR first, then polish it.

Reply:
tracked at https://github.com/apache/incubator-opendal/issues/3426